Formula/pkg-config.rb

```diff
@@ -28,11 +28,10 @@ def install
                           "--with-internal-glib",
                           "--with-pc-path=#{pc_path}"
     system "make"
-    system "make", "check"
     system "make", "install"
   end
 
   test do
-    system "#{bin}/pkg-config", "--libs", "libpcre"
+    system "#{bin}/pkg-config", "--list-all"
```

Comment:
Can we check for something here?

Reply:
The problem is that pkg-config doesn't include any pc files by default and, after this change, there's nothing to read from Homebrew. I'll maybe see if it's possible to create a custom `.pc` file and load it from the testpath.

Reply:
I've got something to work.